ui/00_message/event/st0100/ev015.xml

```diff
@@ -1,13 +1,17 @@
 ﻿<resource>
   <message>
     <original>今度はなんだ？　奴らに聞かせるなよ</original>
+    <translation>Now what? Don't let them hear you.</translation>
   </message>
   <message>
     <original>なんだと！？
 魔物を倒しただって？</original>
+    <translation>Wait, What!?
+Did you defeat the Monster?</translation>
```

Comment:
intentional, but it might not be correct. from the context this was a particular monster that the knights couldnt beat, so I used capitalization to imply that the monster is "special".

Its just some flair, so Ill change it.